pkg/globalnet/controllers/ipam/ipam.go

```diff
@@ -177,8 +177,8 @@ func (i *Controller) processNextObject(objWorkqueue workqueue.RateLimitingInterf
 				}
 
 				// Privileged pods that use hostNetwork will be ignored.
-				if runtimeObj.Status.PodIP == runtimeObj.Status.HostIP {
-					klog.V(log.DEBUG).Infof("Ignoring pod %s on host %s as it uses hostNetworking", runtimeObj.Name, runtimeObj.Status.PodIP)
+				if runtimeObj.Spec.HostNetwork {
+					klog.V(log.DEBUG).Infof("Ignoring pod %s on host %s as it uses hostNetworking", key, runtimeObj.Status.PodIP)
```

Comment:
```suggestion
					klog.V(log.DEBUG).Infof("Ignoring pod %q on host %q as it uses hostNetworking", key, runtimeObj.Status.PodIP)
```

while we are modifying the code.. :) so we are uniform with that in other places.. @tpantelis convinced me.. ':)

Reply:
haha...I'm convinced too ;-)